zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/EngineErrorHandlingTest.java

```diff
@@ -105,20 +105,25 @@ public void setUp() {
     streams.createLogStream(STREAM_NAME, PARTITION_ID);
   }
 
-  @AfterEach
+  @After
   public void tearDown() {
     CloseHelper.close(streamProcessor);
   }
 
   @Test
   public void shouldAutoRejectCommandOnProcessingFailure() {
     // given
+    final MutableLong failingKey = new MutableLong();
+    final MutableLong secondKey = new MutableLong();
     streamProcessor =
         streams.startStreamProcessor(
             STREAM_NAME,
             DefaultZeebeDbFactory.defaultFactory(),
             (processingContext) -> {
               processingState = processingContext.getProcessingState();
+              keyGenerator = processingContext.getProcessingState().getKeyGenerator();
```

Comment:
💭 would an alternative be to reconfigure/wire the processor to use the atomic generator?

Reply:
I think that it's not possible since it does replay and so on, so I'd like to have everything as it's in production code.